extendr-api/src/serializer.rs

```diff
@@ -511,6 +513,36 @@ impl ser::Serialize for Symbol {
     }
 }
 
+impl ser::Serialize for Primitive {
```

Comment:
I guess this isn't the change intentionally included here, right?

Reply:
Yes. I imagine it got sucked into the vortex...

Reply:
I need to finish the `Serialize` work this month for the R foundation milestone but I'm being sucked into a batch of teaching
and project proposals.

Reply:
Oh, that's important.